src/renderer/components/watch-video-playlist/watch-video-playlist.js

```diff
@@ -232,6 +216,23 @@ export default defineComponent({
     }
   },
   methods: {
+    findIndexOfCurrentVideoInPlaylist: function (playlist) {
```

Comment:
nitpick: this should be `videos` / `playlistItems`
`findIndexOfCurrentVideoInPlaylist` is called with various source of video lists (with different sorting orders applied)